Expensify/App
# PR 3: Adding EmojiPicker to the Edit Comment

### Details

- This is the third PR to close the tagged issue. It adds the EmojiPicker to the Edit Comment.  Refer the comment https://github.com/Expensify/App/issues/3245#issuecomment-1025441734

### Fixed Issues

$ https://github.com/Expensify/App/issues/3245

### Tests
1. Tested the EmojiPicker on Composer
2. Tested frequently used emojis
3. Tested skintone for emojis
4. Tested searchInput focus for EmojiPicker on Load
5. Tested scroll for EmojiPicker (had issues with earlier implementation)
6. Closing EmojiPicker doesn't shift the focus to the edit comment
7. EmojiPicker positioning switching between Edit comment and Composer




### QA Steps

1. Go to any chat
2. Select any own message
3. Click on Edit Comment
4. You'll see an Emoji icon and click on the icon
5. Select your emoji, and ensure that it gets entered into the comment.


### Tested On


### Screenshots


#### Web

<img width="1055" alt="web-edit-emoji-picker" src="https://user-images.githubusercontent.com/3069065/152621298-e827577a-e00f-45de-90ac-4924e560fc78.png">

#### Mobile Web

<img width="383" alt="mweb-edit-emoji-picker" src="https://user-images.githubusercontent.com/3069065/152621323-d3aa788e-1a65-4b43-9f55-66d939186bff.png">

#### Desktop

<img width="512" alt="desktop-edit-emoji-picker" src="https://user-images.githubusercontent.com/3069065/152621333-c36b3533-423f-4ba1-a50d-5a1afb32ff2a.png">

#### iOS

<img width="382" alt="ios-edit-emoji-picker" src="https://user-images.githubusercontent.com/3069065/152621691-ba879367-eb1c-4a2a-b4b3-d7721ee1fc81.png">

#### Android

<img width="342" alt="android-edit-emoji-picker" src="https://user-images.githubusercontent.com/3069065/152621708-06273f6d-6c8d-4e8d-9493-f7a0f76ef97c.png">
